### PR TITLE
Adding Forecast Only Model Option 

### PR DESCRIPTION
**Pull request by JessicaMeixner-NOAA**
## Adding forecast model 

A new case for forecast only is added (fcst_only.yaml).  To use this option, follow the examples for 3dvar in the README, except when preparing the workflow substep 4 you will instead do: 
`./setup_case.sh -p HERA ../cases/fcst_only.yaml workflowtest001`

No answers should change as a result of this PR. 

### Issues addressed

- Completes Issue #44 of adding CICE5 and MOM6 to the forecast only workflow.  
- Addresses Issue #84 in main README.md file. 

## Testing

- Both the 3dvar_only and fcst_only cases are currently being tested. 
- Tests are on hera with default modules. 
- Changes are not tested by ctests.

## Dependencies

- None other than waiting for tests to complete.

### CODE REVIEW

File: scripts/prep_forecast.sh [hunk 1]
```diff
@@ -0,0 +1,502 @@
...
+# https://github.com/NOAA-EMC/NEMSdatm/wiki/DATM-Input-File-Descriptions #
+######################################################################
+######################################################################
+
+######################################################################
+# 2.1 input.nml                                                      #
+######################################################################
+
+#Input.nml is an FMS file used by both FV3 and MOM6 
+echo "creating inpu.nml"
```
Comment (guillaumevernieres) [inline scripts/prep_forecast.sh]: typo

File: scripts/prep_forecast.sh [hunk 2]
```diff
@@ -0,0 +1,502 @@
...
+#  restart_interval_write=0
+#  confignamevarfornems="medcold_atm_ocn_ice"
+#  medcoldstart=true    
+#else
+  restart_interval_write=${restart_interval:-86400}    # Interval in seconds to write restarts
+  confignamevarfornems="med_atm_ocn_ice"
+  medcoldstart=false
+#fi
+
+#Caulculate bounds based on resource PETS
```
Comment (guillaumevernieres) [inline scripts/prep_forecast.sh]: typo :point_up:

File: workflow/cases/fcst_only.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+case:
+  settings:
+    SDATE: 2011-10-01t00:00:00
+    EDATE: 2011-10-03t00:00:00
```
Comment (guillaumevernieres) [inline workflow/cases/fcst_only.yaml:4]: Do we need 3 cycles? Would 2 be enough for testing?

**Comment by guillaumevernieres** [comment]
@JessicaMeixner-NOAA : develop was just updated with @flampouris 's work. Could you merge develop into your branch?

File: workflow/cases/fcst_only.yaml (hunk 3 above)
Comment (JessicaMeixner-NOAA) [reply workflow/cases/fcst_only.yaml:4]: That's a good question... For the first first round I thought 3, but 2 in the future is probably sufficient.

File: scripts/prep_forecast.sh (hunk 2 above)
Comment (JessicaMeixner-NOAA) [reply scripts/prep_forecast.sh]: fixed in 6917adc

File: scripts/prep_forecast.sh (hunk 1 above)
Comment (JessicaMeixner-NOAA) [reply scripts/prep_forecast.sh]: fixed in 6917adc

**Review by guillaumevernieres — APPROVED** [review]
Nice and clean, thanks @JessicaMeixner-NOAA :)

**Pull request merged**